### DISCUSSION TITLE
the HTML attribute reference page

### DISCUSSION THREAD

**Discussion by YecineM** [mdn/mdn-community#528] [Content] [answered]
Hello people !
In the HTML attribute reference of MDN, the elements for the **type** attribute should include **ol**.
Please correct it, as I'm not a member of MDN.

**Comment by hamishwillee** [comment, marked as answer]
Hi @YecineM 
Can you please create an issue for this, providing all the information requried by the template: https://github.com/mdn/content/issues

Right now I have no idea what page you are referring to - whereas if you create an issue or a pull request then we can see exactly what you mean, because you will need to provide us the URL of the page and other information.
